### PR TITLE
Add method for lookup by id

### PR DESCRIPTION
**Pull request by rabidaudio**
The `fetch` method can be dangerous since there can be more than one item with the same title, so I needed an id lookup method for my project.

Here it is under `find` if you want it.

### CODE REVIEW

File: lib/omdb/api.rb [hunk 1]
```diff
@@ -32,6 +32,22 @@ def fetch(title, year = "", tomatoes = false, plot = "short")
...
     end
+    
+    # fetches a movie by IMDB id
+    # set tomatoes to true if you want to get the rotten tomatoes ranking
+    # set plot to full if you want to have the full, long plot
+    def find(id, tomatoes = false, plot = "short")
+      res = network.call({i: id, tomatoes: tomatoes, plot: plot})
+      
+      if res[:data]["Response"] == "False"
+        response = {:status => 404}
```
Comment (jvanbaarsen) [inline lib/omdb/api.rb]: Can you change this to ruby 1.9 syntax: `response = { status: 404 }`?

**Comment by jvanbaarsen** [comment]
@rabidaudio Thanks! Can you please also add a test for this feature?

**Comment by rabidaudio** [comment]
added tests and fixed the hash syntax (also in the place I copied it from)

**Comment by jvanbaarsen** [comment]
@rabidaudio Thanks! :dancers:

**Pull request merged**